### DISCUSSION TITLE
vela workflow resume  只能通过 cli command 才能操作吗？有api 操作方式吗？或者k8s java client api 操作方式？

### DISCUSSION THREAD

**Discussion by skyshaokai** [kubevela/kubevela#3993] [Q&A] [answered]
https://kubevela.net/zh/docs/cli/vela_workflow
vela workflow resume

**Comment by wonderflow** [comment, marked as answer]
`vela workflow resume` is a simple wrapper to change the application status.

You can manually change the application status like below:


```
status:
  workflow:
-    suspend: true
+    suspend: false
```

**Reply by skyshaokai** [reply]
I try this method , but not work.

1、kubectl edit  app/first-vela-app
application.core.oam.dev/first-vela-app edited

edited suspend to  false

<img width="444" alt="image" src="https://user-images.githubusercontent.com/11512967/170418535-591e35bc-74a8-4a68-876d-32dc01e8c209.png">

2、kubectl get app/first-vela-app -o yaml

but suspend is still true

<img width="575" alt="image" src="https://user-images.githubusercontent.com/11512967/170418695-c16e54dc-829a-4ef8-b44e-0989bedace97.png">


<img width="810" alt="image" src="https://user-images.githubusercontent.com/11512967/170418624-05e8dd8d-9515-42c8-a74f-62c1f6580d08.png">

**Reply by wonderflow** [reply]
Status is a subresource of k8s, you need to use code or plugin to edit it. https://github.com/kubernetes/kubectl/issues/564
